linera-core/src/unit_tests/wasm_worker_tests.rs

```diff
@@ -207,7 +209,7 @@ where
         messages: vec![OutgoingMessage {
             destination: broadcast_channel,
             authenticated_signer: None,
-            is_skippable: false,
+            attributes: MessageAttributes::skippable(),
```

Comment:
This was `false`; should it be `default()` now?

Reply:
good catch!

Reply:
actually it may be part of ec39152bc8f158dd3272d824ddf9d22806e1760e. Maybe I'll move this commit out of this PR.